doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,293 @@
+
+      #. Run the following command in your terminal:
+
+         .. parsed-literal::
+            :class: highlight
+
+            /Users/*yourusername*/ncs-lcs/nrfutil toolchain-manager launch --shell --chdir "/Users/*yourusername*/ncs-lcs/work-dir" --ncs-version v2.6.99-cs2
+
+         This makes west and other development tools in the |NCS| toolchain environment available in the same shell session.
+
+         .. caution::
+            When working with west in the customer sampling release version of |NCS|, you must always use this shell window.
+
+      #. Run the following commands in your terminal to install the correct lxml dependency:
+
+         .. parsed-literal::
+            :class: highlight
+
+            pip uninstall -y lxml
+            pip install lxml
+
+      #. Install the `Serial Terminal from nRF Connect for Desktop`_.
+
+      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`ncs-lcs` directory, and start over.
+
+      We recommend adding the nRF Util path to your environmental variables.
+
+Installing the |NCS| 2.6.99-cs2
+*******************************
+
+After you have installed the toolchain, complete the following steps to get the |NCS| 2.6.99-cs2:
+
+1. In the terminal window opened during the toolchain installation, initialize west with the revision of the |NCS| from the customer sampling by running the following command:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west init -m https://github.com/nrfconnect/sdk-nrf --mr v2.6.99-cs2
+
+#. Enter the following command to clone the project repositories::
+
+      west update
+
+   Depending on your connection, this might take some time.
+
+#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
+   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
+
+      west zephyr-export
+
+Your directory structure now looks similar to this::
+
+    ncs-lcs/work-dir
+    |___ .west
+    |___ bootloader
+    |___ modules
+    |___ nrf
+    |___ nrfxlib
+    |___ zephyr
+    |___ ...
+
+Note that there are additional folders, and that the structure might change.
+The full set of repositories and folders is defined in the manifest file.
+
+Updating the Terminal application
+*********************************
+
+To update `nRF Connect Serial Terminal`_, follow these steps:
+
+1. On your computer, open `nRF Connect for Desktop`_
+   If there is an update available, a pop up will notify you of its availability.
+#. If available, install the update from the pop up screen.
+#. Update `nRF Connect Serial Terminal`_ from the nRF Connect for Desktop application)
+
+If you are using the nRF Terminal application part of the `nRF Connect for Visual Studio Code`_ extension, open Visual Studio Code instead and ensure you are running the newest version of both the editor and the extension.
```

Comment:
Is this section needed? I don't think any other migration guides ever mention needing to update the tools we provide.

Reply:
This is a special case, it will guide users to update all the tools as they come from nrf-next.